package.json

```diff
@@ -12,10 +12,10 @@
   ],
   "author": "Remind 101 (https://github.com/remind101)",
   "dependencies": {
-    "generic-pool": "^2.0.4",
-    "q": "^1.0.1",
-    "redis": "^0.10.3",
-    "underscore": "^1.6.0"
+    "generic-pool": "~2.4",
+    "q": "~1.0",
+    "redis": "2.6.2",
```

Comment:
the redis lib supports url parsing since v1.0.0, we could leverage that (right now we have our own implementation in this lib)